web/source/keymanweb.js

```diff
@@ -1711,13 +1711,13 @@
       e = window.event;
       if(!e)
       {
-        e = keymanweb._GetLastActiveElement();
-        if(e)
+        var e2 = keymanweb._GetLastActiveElement();
+        if(e2)
         {
-          e = e.ownerDocument;
```

Comment:
Since we're going ahead and introducing a new variable, can we call it `elem` instead of `e2`? As the types are different anyway! Also, if you could `{}`, per new convention, any changed code, that would be great.